spec/models/manageiq/providers/amazon/regions_spec.rb

```diff
@@ -15,4 +15,19 @@
       expect(described_class.names).not_to include('us-gov-west-1')
     end
   end
+
+  context "adds additional regions from settings" do
+    it "includes regions from additional_regions" do
+      custom_region = {
+        "my-custom-region" => {
+          :name        => "My Custom Region",
+          :hostname    => "localhost",
+          :description => "Custom Region"
+        }
+      }
+
+      allow(Settings.ems.ems_amazon).to receive(:additional_regions).and_return(custom_region)
```

Comment:
Additonally, `additional_regions` will be an array, but this inserts a single hash instead.